src/mutator.ts

```diff
@@ -17,9 +17,10 @@ export default function mutator<TAction extends ActionMessage, TReturn>(
     }
 
     // Wrap the callback in a MobX action so it can modify the store
-    let wrappedTarget = action(getPrivateActionType(actionCreator), (actionMessage: TAction) => {
+    const actionType = getPrivateActionType(actionCreator);
+    let wrappedTarget = action(actionType, (actionMessage: TAction) => {
         try {
-            getGlobalContext().currentMutator = actionCreator.name;
+            getGlobalContext().currentMutator = actionType;
```

Comment:
Is this really what you want?  The private action type is just a unique ID (a number) that may not even be the same from one session to another.  Why wouldn't you want the action name?

Reply:
Never mind, I was thinking of action *ID*, this is action *type*.  This makes sense.